tests/helpers.py

```diff
@@ -112,16 +112,19 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 def path_to_executable(program_name):
+    path = "."
     if sys.platform.startswith("win"):
         if 'APPVEYOR_BUILD_FOLDER' not in os.environ: os.environ['APPVEYOR_BUILD_FOLDER'] = "."
-        return os.path.join(
-            os.environ['APPVEYOR_BUILD_FOLDER'],
-            'build',
-            'tests',
-            program_name + ".EXE"
-        )
+        path = os.path.join(path, os.environ['APPVEYOR_BUILD_FOLDER'])
+    path = os.path.join(path, "build", "tests")
+    for executable in [
+        os.path.join(path, program_name),
+        os.path.join(path, program_name + ".EXE"),
+        os.path.join(path, program_name + ".exe")]:
+            if os.path.isfile(executable):
+                return executable
     else:
```

Comment:
Is the indentation of this `else` weird? In fact I think maybe the else should be eliminated and then the function flows into the `assert False` if there was no `return executable` completed...

Reply:
@dstebila Fixed it